### PR TITLE
feat: 🎸 Move `createPortfolio` to `Identities` namespace

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
BREAKING CHANGE: 🧨 `Portfolios.create` has been removed in favour of
`Identities.createPortfolio`

### CODE REVIEW

File: src/Identities.ts [hunk 1]
```diff
@@ -33,6 +45,11 @@ export class Identities {
    */
   public registerIdentity: ProcedureMethod<RegisterIdentityParams, Identity>;
 
+  /**
+   * Create a new Portfolio under the ownership of the Current Identity
```
Comment (polymath-eric) [inline src/Identities.ts:49]: (nit)
```suggestion
   * Create a new Portfolio under the ownership of the signing Identity
```
I think this makes it a bit more clear compared to "Current".

**Review by polymath-eric — APPROVED** [review]
LGTM

**Review by monitz87 — APPROVED** [review]

**Pull request merged**